composer.json

```diff
@@ -17,23 +17,22 @@
         "psr/cache": "^1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "5.6.*"
+        "phpunit/phpunit": "^5.6",
+        "php-coveralls/php-coveralls": "^2.1"
```

Comment:
Adding the `php-coveralls` inside `require-dev` block to let CI build be successful.